superduperdb/ext/llm/base.py

```diff
@@ -94,6 +93,10 @@ def get_kwargs(self, func, *kwargs_list):
                 new_kwargs[k] = v
         return new_kwargs
 
+    @property
+    def prompter(self):
```

Comment:
does it make sense to cache it?
WDYT? @jieguangzhou

Reply:
The reason for this modification is that we can dynamically modify prompt_template, and the prompter will change in real time. In fact, its initialization time can be ignored.